onnx/helper.py

```diff
@@ -133,7 +134,15 @@ def make_attribute(key, value):
         if all(isinstance(v, float) for v in value):
             attr.floats.extend(value)
         elif all(isinstance(v, numbers.Integral) for v in value):
-            attr.ints.extend(value)
+            # Turn np.int64 into built-in numpy.int64.
+            # In some version of Python 3, numpy.int64 is not subtype of built-in int.
+            new_value = []
+            for v in value:
+                if isinstance(v, np.int64):
```

Comment:
We can probably blindly do `attrs.ints.extend(int(v) for v in value)`.

btw. Did some googling and found the related discussion here: https://github.com/numpy/numpy/issues/2951. Turns out in Python3, none of the numpy integers are instances of Python native `int`, and starting from numpy 1.9.0, all of the numpy integers are instances of `numbers.Integral`